### PR TITLE
fix(source): remove signing_key option on webhook

### PR DESCRIPTION
**Pull request by mdeltito**
The `signing_key` option is no longer managed at this layer.

Ref: LOG-18872

### CODE REVIEW

File: internal/provider/models/sources/test/datadog_test.go [hunk 1]
```diff
@@ -20,9 +20,7 @@ func TestDatadogSource(t *testing.T) {
 					resource "mezmo_pipeline" "test_parent" {
 							title = "parent pipeline"
 						}`) + `
-					resource "mezmo_datadog_source" "my_source" {
-						signing_key = "nope"
-					}`,
+					resource "mezmo_datadog_source" "my_source" {}`,
```
Comment (jorgebay) [inline internal/provider/models/sources/test/datadog_test.go:23]: Should we delete the parameter from `internal/provider/models/sources/datadog.go` too?

**Review by jorgebay — COMMENTED** [review]
This is technically a breaking change, thought I doubt anyone is using it this way.

File: internal/provider/models/sources/test/datadog_test.go (hunk 1 above)
Comment (mdeltito) [reply internal/provider/models/sources/test/datadog_test.go:23]: it's not actually part of that model, looks like just copypasta

Comment (jorgebay) [reply internal/provider/models/sources/test/datadog_test.go:23]: oh! nice cleanup!

**Review by jorgebay — APPROVED** [review]

**Comment by jorgebay** [comment]
I'll land this as the build is broken with the latest version of the service.

**Pull request merged**